http-clients-api/src/main/java/com/palantir/remoting/clients/ClientConfig.java

```diff
@@ -87,13 +86,13 @@ public static ClientConfig fromServiceConfig(ServiceConfiguration serviceConfig)
 
         // timeouts
         if (serviceConfig.connectTimeout().isPresent()) {
-            clientConfig.connectTimeout(serviceConfig.connectTimeout().get().toMilliseconds(), TimeUnit.MILLISECONDS);
+            clientConfig.connectTimeout(serviceConfig.connectTimeout().get());
```

Comment:
you have to write this code somewhere, right? not sure what the better place is?